### PR TITLE
[13.0][MIG] stock_account_change_qty_reason

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module stock_account_change_qty_reason to v13.

Depends on:
- [x] stock_change_qty_reason #813 

cc ~ @ForgeFlow

### CODE REVIEW

File: stock_account_change_qty_reason/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1 +1 @@
-* Héctor Villarreal <hector.villarreal@eficent.com>
+* Héctor Villarreal <hector.villarreal@forgeflow.com>
```
Comment (LoisRForgeFlow) [inline stock_account_change_qty_reason/readme/CONTRIBUTORS.rst:1]: Add yourself

**Review by LoisRForgeFlow — APPROVED** [review]
Code review, LGTM

File: stock_account_change_qty_reason/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (AdriaGForgeFlow) [reply stock_account_change_qty_reason/readme/CONTRIBUTORS.rst:1]: Yes, sir

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**